site/includes/follow.hbs

```diff
@@ -1,4 +1,8 @@
+{{#contains "theme, topic" page.pageType }}
 <section class="followus">
+{{else}}
+<section class="followus pull-right">
```

Comment:
Be carefull when using pull-* That prevent the AEM editor to edit the pull-* component. 

Also, after look which pages that are including ```follow``` include, some already have a ```pull-right``` set on the parent. If you are going to add ```pull-right``` in the include, then go on every template that use that include and remove the ```pull-right``` on them.

When using pull-* take in consideration is the following structure can be used
```
<div class="row">
   <div class="col-sm-4 pull-right">
      ... follow us....
   </div>
   <div class="col-sm-auto">
      ... rest of the content ...
   </div>
</div>
```

Reply:
Here an alternative for the follow us

```xml
<div class="col-sm-5 pull-right">
   <div class="row">
      <div class="col-md-auto pull-right col-sm-12">
         <section class="followus">....</section>
         <section id="facebook">....</section>
         <section id="youtube">....</section>
      </div>
   </div>
</div>
```